.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: go
 go:
-- 1.7
-- 1.8
-- 1.9
-- tip
+- "1.7"
+- "1.8"
+- "1.9"
+- "1.10"
```

Comment:
These versions have to be quoted now, because the unquoted `1.10` will be interpreted as Go `1.1`.